tests/aws/services/cloudformation/resources/test_dynamodb.py

```diff
@@ -163,3 +163,45 @@ def test_ttl_cdk(aws_client, snapshot, infrastructure_setup):
         table_name = outputs["TableName"]
         table = aws_client.dynamodb.describe_time_to_live(TableName=table_name)
         snapshot.match("table", table)
+
+
+@markers.aws.validated
+@markers.snapshot.skip_snapshot_verify(
+    [
+        "$..Table.ProvisionedThroughput.LastDecreaseDateTime",
+        "$..Table.ProvisionedThroughput.LastIncreaseDateTime",
+        "$..Table.Replicas",
```

Comment:
I'd encourage adding a comment why these are skipped. Is it because LocalStack returns those keys but AWS doesn't?

Reply:
We return them, but AWS doesn't. I'll add the comment in the code